index.bs

```diff
@@ -703,34 +751,74 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator with which to complete the operation.
 </div>
 
+### <dfn interface>AuthenticatorResponse</dfn> interfaces ### {#iface-authenticatorresponse}
 
-## Information about Scoped Credential (interface <dfn interface>ScopedCredentialInfo</dfn>) ## {#iface-credentialInfo}
+The {{AuthenticatorResponse}} interface and its derieved interfaces represent the data 
 
-<pre class="idl">
-    [SecureContext]
-    interface ScopedCredentialInfo {
-        readonly    attribute ArrayBuffer   clientDataJSON;
-        readonly    attribute ArrayBuffer   attestationObject;
-    };
-</pre>
+<xmp class="idl">
+  [SecureContext]
+  interface AuthenticatorResponse {
+    readonly attribute ArrayBuffer clientDataJSON;
+  };
+</xmp>
+<dl dfn-for="AuthenticatorResponse">
+    :   <dfn attribute>clientDataJSON</dfn>
+    ::  This attribute contains a representation of the data sent from the client to the authenticator in order to generate
+        a given response. The {{AuthenticatorAttestationResponse}} and {{AuthenticatorAssertionResponse}} 
```

Comment:
Thanks! Fixed in https://github.com/w3c/webauthn/pull/384/commits/fedccedb468282665936264105fac6ee09e577cf.